plugins/woocommerce/includes/admin/class-wc-admin-webhooks.php

```diff
@@ -45,7 +45,7 @@ public function allow_save_settings( $allow ) {
 	 */
 	private function is_webhook_settings_page() {
 		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-		return isset( $_GET['page'], $_GET['tab'], $_GET['section'] ) && 'wc-settings' === $_GET['page'] && 'advanced' === $_GET['tab'] && 'webhooks' === $_GET['section'];
+		return is_wc_admin_settings_page() && isset( $_GET['tab'], $_GET['section'] ) && 'advanced' === $_GET['tab'] && 'webhooks' === $_GET['section'];
```

Comment:
Note: same as original, but adds `is_admin()` check, which looks safe based on the usage context.